src/snippets/track-posted-snippets.ts

```diff
@@ -0,0 +1,29 @@
+import { writeFileSync, readFileSync } from 'fs';
+import path from 'path';
+
+const TRACKED_FILE = path.join(process.cwd(), 'previous-trending-snippets.json');
+
+export function loadPostedSnippets(): string[] {
+  try {
+    console.log('[TRACK-POSTED-SNIPPETS] Loading posted snippets...');
```

Comment:
```suggestion
```

Remove the console logs from here and other places, if you want to debug. Use the `debug` package